### PR TITLE
README: added info about Debian package

### PR DESCRIPTION
**Pull request by spk**
Just some info about install on Debian.
Cheers

### CODE REVIEW

**Comment by agentzh** [comment]
Thank you for contributing, but I am afraid I don't want to add non-official Debian packages to the official installation documentation.  Thanks for your understanding.

**Comment by spk** [comment]
It is an official Debian package sponsored by a Debian developer.

File: README.markdown [hunk 1]
```diff
@@ -568,6 +570,12 @@ and then load the library in Lua:
     local server = require "resty.websocket.server"
 ```
 
+if you are using Debian package:
+
+```lua
+    local server = require "nginx.websocket.server"
```
Comment (agentzh) [inline README.markdown]: Sorry, I hate the way the package changes the Lua module name, which is breaking API compatibility.

Comment (spk) [inline README.markdown]: I can understand that, I fellow the naming from Debian. Thank for you feedback.

**Pull request closed without merging**